src/dbg/taskthread.cpp

```diff
@@ -0,0 +1,95 @@
+#include <Windows.h>
```

Comment:
This is the include that gives the warnings. If you remove it everything works as desired (without modifying `_global.h`).